common/config/rush/pnpm-lock.yaml

```diff
@@ -102,28 +102,28 @@ importers:
     devDependencies:
       '@azure-tools/test-recorder': 3.0.0
       '@azure/abort-controller': 1.1.0
-      '@azure/core-util': 1.4.0
+      '@azure/core-util': 1.5.0
       '@azure/core-xml': 1.3.4
-      '@microsoft.azure/autorest.testserver': 3.3.47
-      '@types/chai': 4.3.5
-      '@types/chai-as-promised': 7.1.5
+      '@microsoft.azure/autorest.testserver': 3.3.48
```

Comment:
This is the reason why we the changes in `packages/autorest.typescript` folder.

Reply:
the example key in testserver 3.3.47 version is string_putNull
```
        "x-ms-examples": {
          "string_putNull": {
            "$ref": "./examples/string_getBase64Encoded.json"
          }
        },
```
then changed into string_getBase64Encoded in version 3.3.48
```
       "x-ms-examples": {
          "string_getBase64Encoded": {
            "$ref": "./examples/string_getBase64Encoded.json"
          }
        },
```